auto/dev-environment

```diff
@@ -7,10 +7,11 @@ trap "docker-compose down --volumes --remove-orphans" 0
 
 # Create volume
 docker volume create --name ivy-cache > /dev/null
+docker volume create --name coursier-cache > /dev/null
 docker volume create --name docker-scala-sbt-cache > /dev/null
```

Comment:
given that this change upgrades sbt and starts using coursier the `docker-scala-sbt-cache` becomes unused, so probably should be removed as part of this PR.

Reply:
Removed.  Thanks @felipeeflores